.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,25 @@
+on:
+  push:
+    branches:
+      - release/v*
+
+jobs:
+  bump-version:
+    name: Bump provider version in docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Load secrets
+        uses: secrethub/actions/env-export@v0.1.0
+        env:
+          SECRETHUB_CREDENTIAL: ${{ secrets.SECRETHUB_CREDENTIAL }}
+          GITHUB_TOKEN: secrethub://secrethub/terraform-provider/release/github-token
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          token: ${{ env.GITHUB_TOKEN }}
+      - name: Bump version
+        uses: florisvdg/action-version-bump@v0.1.0
+        with:
+          sed: 's/\(version = "\).*"/\1$VERSION"/g'
+          file: docs/index.md
+          author_email: bender.github@secrethub.io
```

Comment:
What's the purpose of this now?

Reply:
I've removed this in the next commit: fb630bf

This commit was from before I applied your suggestion of referring to the button.

Reply:
Ah. Looked over that because the whole file was deleted. :+1: in that case.